public/app/features/explore/RichHistory/RichHistoryStarredTab.test.tsx

```diff
@@ -9,19 +9,37 @@ import { RichHistoryStarredTab, Props } from './RichHistoryStarredTab';
 
 jest.mock('../state/selectors', () => ({ getExploreDatasources: jest.fn() }));
 
-const setup = (propOverrides?: Partial<Props>) => {
+jest.mock('@grafana/runtime', () => ({
+  ...(jest.requireActual('@grafana/runtime') as unknown as object),
```

Comment:
```suggestion
  ...jest.requireActual('@grafana/runtime'),
```
same as before :)

Reply:
Fixed in 01d3c34eee6438605d49e11f2337147bf4887ae2